docs/source/concept_models/vivarium_swissre_cervicalcancer/cervical_cancer_concept_model.rst

```diff
@@ -0,0 +1,597 @@
+ - csmr_432: 432_ets_deaths
+ 
+ Zach is working on updating forecast data to GBD 2019, so expect these filenames 
+ to change soon. 
+
+.. _5.3:
+
+5.3 Models
+----------
+
+.. _5.3.1:
+
+5.3.1 Core cervical cancer model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+see :ref:`cervical cancer cause model<2017_cancer_model_cervical_cancer>`
+
+.. _5.3.2:
+
+5.3.2 Screening and detection model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:underline:`I. Screening algorithm`
+
+Cervical cancer screening algorithm was determined by three variables 
+ 1) Sex
+ 2) Age 
+ 3) Diagnosis of HPV infection
+We will assign every simulant a cytology plus HPV co-test at the beginning of 
+the simulation.
+
+.. image:: cervical_screening_branches.svg 
+
+.. list-table:: Screening branches
+   :header-rows: 1
+
+   * - Branch
+     - Sex
+     - Age
+     - Screening technology
+     - Screening result
+     - proportion
+     - follow-up test
+     - Sensitivity
+     - Specificity
+   * - A
+     - Female
+     - 25-65
+     - Cytology plus HPV test
+     - HPV negative, Cytology negative
+     - 17.4%
+     - Cytology, every 3 years
+     - 65.9% with 95%CI 54.9 to 75.3
+     - 100%
+   * - B
+     - Female
+     - 25-65
+     - Cytology plus HPV test
+     - HPV positive, Cytology negative
+     - 23.5%
+     - Cytology, every year
+     - 65.9% with 95%CI 54.9 to 75.3
+     - 100%
+   * - C
+     - Female
+     - 25-65
+     - Cytology plus HPV test
+     - HPV negative, Cytology positive
+     - 5.9%
+     - Colposcopy
+     - 
+     - 
+   * - D
+     - Female
+     - 25-65
+     - Cytology plus HPV test
+     - HPV positive, Cytology positive
+     - 53.2%
+     - Colposcopy
+     - 
+     - 
+   * - E
+     - Female
+     - <25 or 65+
+     - No screening
+     - 
+     - 
+     - 
+     - 
+     - 
+   * - F
+     - Male
+     - Any
+     - No screening
+     - 
+     - 
+     - 
+     - 
+     - 
+
+In initialization, We assume that
+ - No one has prior knowledge of their disease status (and HPV status) on day one 
+   of the simulation.
+ - All simulants are buying insurance on day one of the simulation.
+ - For simulants in cervical cancer (CC) state regardless of detection, they have 
+   a transition rate of 0.1 (per person-year) of moving into a recovered (R) state; this results in an average duration in state CC of 10 years. People in state CC and R follow exactly the same screening algorithm, namely branch A, B, C, or D depending on their age, sex, and co-test result. Simulants do not ever make a second cervical cancer claim, therefore the negative screening results were expected for those in R state in order to avoid double counting the CI claim from detected cervical cancer.
+ - The initial age of cervical cancer screening is set to 30 years old since HPV 
+   co-test (cytology plus HPV test) is not recommended for women under 30 according to guidelines from American Cancer Society and U.S. Preventive Services Task Force.
+
```

Comment:
Highlighting the difference in numbers between the table and written description again - starting screening at age 25 in table, but at age 30 in written description and tree diagram. We need to stay consistent to avoid confusion when implementing on the SE side. 

I also think it might be worth having an RT discussion on how to go about HPV screening and co-testing (maybe reach out to Manoj with specific questions to get a better understanding of how this is implemented in practice). Women typically start getting screened for cervical cancer starting at age 21, so pushing screening to begin at 30 in order to factor in co-testing could have some big implications for this model (if we stick with this we need to think through the limitations).

Reply:
I noticed the age confusion here, the starting age for cervical cancer screening is 25 years old from Chinese guideline (21 from CDC recommendations). 30 is used for co-test but i would like to discuss with cancer expert and software engineers on this.